packages/curvenote-cli/src/submissions/submit.ts

```diff
@@ -214,9 +217,16 @@ export async function submit(session: ISession, venue: string, opts?: SubmitOpts
       ...job.results,
     });
 
-    // const cdnKey = 'ad7fa60f-5460-4bf9-96ea-59be87944e41'; // dev debug
     const cdn = opts?.draft ? session.TEMP_CDN : session.PRIVATE_CDN;
-    const { cdnKey } = await uploads.uploadToCdn(session, cdn, opts);
+    let cdnKey: string;
+    if (!process.env.DEV_CDN || process.env.DEV_CDN === 'false') {
+      const uploadResult = await uploads.uploadToCdn(session, cdn, opts);
+      cdnKey = uploadResult.cdnKey;
+    } else if (process.env.DEV_CDN.match(CDN_KEY_RE)) {
```

Comment:
This makes the previously commented-out `dev debug` more explicit.

If you do not want to hit the cdn, you may use default content with `DEV_CDN=true curvenote submit [venue]`

Or you can specify other existing content with `DEV_CDN=my-existing-cdn-uuid curvenote submit [venue]`